Sources/ApolloCodegenLib/Frontend/JavaScript/rollup.config.js

```diff
@@ -10,7 +10,7 @@ const options = {
     file: "../dist/ApolloCodegenFrontend.bundle.js",
     format: "iife",
     name: "ApolloCodegenFrontend",
-    sourcemap: true,
+    sourcemap: false,
```

Comment:
Does this remove the `ApolloCodegenFrontend.bundle.js.map` file? It looks like the file is still in the `dist` directory but empty. 😕

Reply:
^that explains the answer to one of my questions, but now I have a new one: Why are we turning this off?

Reply:
Do we need a source map? It's possible `true` is the default for that value in Rollup.

Reply:
Yeah that's what it does. The file was always empty, which is why I'm removing it. You're saying it's not being removed? My commit shows it deleted.
![image](https://user-images.githubusercontent.com/6243461/145308031-8cecf91a-bf8d-48ba-8f1d-0bf58072ea3a.png)

Reply:
Yes you're correct, it is being deleted. I didn't look at the change properly. 👍🏻

Reply:
> The file was always empty

It had content in before this deletion. I don't believe it was ever used though.